src/main/java/uk/gov/hmcts/darts/audio/mapper/GetAudioRequestResponseMapper.java

```diff
@@ -25,9 +25,9 @@ public GetAudioRequestResponseV1 mapToAudioRequestSummary(EnhancedMediaRequestIn
         response.setMediaRequestStatus(MediaRequestStatus.fromValue(enhancedMediaRequestInfo.getMediaRequestStatus().toString()));
         response.setLastAccessedTs(transformedMedia.getLastAccessed());
         response.setOutputFilename(transformedMedia.getOutputFilename());
-        String outputFilename = transformedMedia.getOutputFormat();
+        uk.gov.hmcts.darts.audio.enums.AudioRequestOutputFormat outputFilename = transformedMedia.getOutputFormat();
```

Comment:
Please can you rename this variable - it doesn't make sense.
Maybe it should be `outputFormat`?

Reply:
done